packages/xo-web/src/xo-app/sr/tab-advanced.js

```diff
@@ -79,7 +79,7 @@ export default ({ sr }) => (
           <tbody>
             <tr>
               <th>{_('provisioning')}</th>
-              <td>{defined(sr.allocationStrategy, _('noValue'))}</td>
+              <td>{defined(sr.allocationStrategy, _('unknown'))}</td>
```

Comment:
This could have been an other PR or at least a separate commit, because it's not related to the title of this PR.

I'll accept it for this time.